lua/hlslens/main.lua

```diff
@@ -36,11 +36,18 @@ local function autocmd(initial)
     if not initial then
         cmd([[
             aug HlSearchLens
-                au CursorMoved * lua require('hlslens.main').refresh()
+                au CursorMoved,CursorMovedI * lua require('hlslens.main').refresh()
```

Comment:
Would you move `CursorMovedI` event and action to line 48? If `calm_down = false`, `CursorMovedI` is redundant.

Reply:
Ah, I see now. Fixed.